src/sentry/db/router.py

```diff
@@ -122,14 +121,14 @@ def _resolve_silo_connection(self, silo_modes: Iterable[SiloMode], table: str) -
         else:
             return None
 
-    def _find_model(self, table: str, app_label: str) -> Model | None:
+    def _find_model(self, table: str) -> type[Model] | None:
         # Use django's model inventory to find our table and what silo it is on.
-        for model in apps.get_models(app_label):
+        for model in apps.get_models(include_auto_created=True):
```

Comment:
the type checker pointed out that this call was nonsense (passing a string into a boolean argument) -- I opted to keep the same behaviour (since the string was always truthy)